docs/pages/versions/unversioned/sdk/light-sensor.md

```diff
@@ -0,0 +1,165 @@
+import { LightSensor } from 'expo-sensors';
+
+export default function App() {
+  const [luminosity, setLuminosity] = useState(null);
+
+  useEffect(() => {
+    _toggle();
+  }, []);
+
+  useEffect(() => {
+    return () => {
+      _unsubscribe();
+    };
+  }, []);
+
+  const _toggle = () => {
+    if (this._subscription) {
+      _unsubscribe();
+    } else {
+      _subscribe();
+    }
+  };
+  
+  const _subscribe = () => {
+    this._subscription = LightSensor.addListener(lightSensorData => {
+      setLuminusity(lightSensorData.illuminance);
+    });
+  };
+
+  const _unsubscribe = () => {
+    this._subscription && this._subscription.remove();
+    this._subscription = null;
+  };
+
+  return (
+    <View style={styles.sensor}>
+      <Text>Light Sensor:</Text>
+      <Text>Luminosity: {Platform.OS === 'android' ? `${luminosity} lx` : `Only available on Android`}</Text>
+      <View style={styles.buttonContainer}>
+        <TouchableOpacity onPress={_toggle} style={styles.button}>
+          <Text>Toggle</Text>
+        </TouchableOpacity>
+      </View>
+    </View>
+  );
+}
+
+/* @hide const styles = StyleSheet.create({ ... }); */
+const styles = StyleSheet.create({
+  buttonContainer: {
+    flexDirection: 'row',
+    alignItems: 'stretch',
+    marginTop: 15,
+  },
+  button: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    backgroundColor: '#eee',
+    padding: 10,
+  },
+  sensor: {
+    marginTop: 45,
+    paddingHorizontal: 10,
+  },
+});
+/* @end */
+```
+
+</SnackInline>
+
+## API
+
+```js
+import { LightSensor } from 'expo-sensors';
+```
+
+## Methods
+
+### `LightSensor.isAvailableAsync()`
+
+> You should always check the sensor availability before attempting to use it.
+
+Returns a promise which resolves into a boolean denoting the availability of the device light sensor.
+
+| OS      | Availability                |
+| ------- | --------------------------- |
+| iOS     | `N/A`                       |
+| Android | Android 2.3+ (API Level 9+) |
+| Web     | `N/A`                       |
+
+#### Returns
+
+- A promise that resolves to a `boolean` denoting the availability of the sensor.
+
+### `LightSensor.addListener((data: LightSensorMeasurement) => void)`
+
+Subscribe for updates to the light sensor.
+
+```js
+const subscription = LightSensor.addListener((luminosity) => {
+  console.log(luminosity.illuminance);
+});
+```
+
+#### Arguments
+
+- **listener (_function_)** -- A callback that is invoked when a LightSensor update is available. When invoked, the listener is provided a single argument that is the luminosity value.
+
+#### Returns
+
+- A subscription that you can call `remove()` on when you would like to unsubscribe the listener.
+
+### `LightSensor.removeAllListeners()`
+
+Removes all listeners.
+
+## Types
+
+### `LightSensorMeasurement`
+
+```typescript
+type LightSensorMeasurement = {
+  luminosity: number;
+};
```

Comment:
This type isn't up-to-date, right?
Please decide if you want to use `luminosity` or `illuminance`.